app.js

```diff
@@ -37,3 +38,147 @@ routes.init(app);
 app.use(errors.handle);
 
 module.exports = app;
+
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+// ---------
+const request = require('request');
+
+// This URL should return JSON data
+const url = 'https://jsonplaceholder.typicode.com';
+
+const PORT = process.env.PORT || 4001;
+
+// Use static server to serve the API
+app.use(express.static('public'));
+
+const photosRouter = express.Router();
+app.use('/photos', photosRouter);
+
+const albumsRouter = express.Router();
+app.use('/albums', albumsRouter);
+
+// `app.get()` call:
+app.get('/', (req, res) => {
+  console.log('/');
+  // Here we would send back the moods array in response
+  // console.log(req);
+  // `Welcome, the requested data will be printed next: ${
+  // res.json(gbody);
+  res.send('Do you want to see albums, photos? Specify that in the endpoint!');
+});
+
+// We show all the albums:
+albumsRouter.get('/', (req, res) => {
+  console.log('----/albums----');
+
+  request(`${url}/albums`, (error, response, body) => {
+    // Print the error if one occurred
+    console.log('error:', error);
```

Comment:
You should do this conditionally. 

`if (err) // SOme action to handle the error like send a propper error response`